### PR TITLE
[MySql] Add parameter MinimalTlsVersion for the cmdlets.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

New/Update-AzMySqlServer is missing parameter for minimal-tls-version 

Fix: #13170

### CODE REVIEW

File: src/MySql/custom/New-AzMySqlServer.ps1 [hunk 1]
```diff
@@ -177,6 +184,15 @@ function New-AzMySqlServer {
               $Parameter.SslEnforcement = [Microsoft.Azure.PowerShell.Cmdlets.MySql.Support.SslEnforcementEnum]::Enable
           }
 
+          if ($PSBoundParameters.ContainsKey('MinimalTlsVersion')) {
+              $Parameter.MinimalTlsVersion = $PSBoundParameters['MinimalTlsVersion']
+              $null = $PSBoundParameters.Remove('MinimalTlsVersion')
+          }
+          else
+          {
+              $Parameter.MinimalTlsVersion = [Microsoft.Azure.PowerShell.Cmdlets.MySql.Support.MinimalTlsVersionEnum]::TlsEnforcementDisabled
```
Comment (msJinLei) [inline src/MySql/custom/New-AzMySqlServer.ps1:193]: MinimalTlsVersion is optional. Why assign a default value to it?

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/MySql/resources/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Resources
```
[the remark this answers is not in the document]
Comment (LucasYao93) [reply src/MySql/resources/readme.md]: Done

File: src/MySql/test/Az.MySql-TestResults.xml [hunk 3]
```diff
@@ -1,400 +1,400 @@
 ﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```
[the remark this answers is not in the document]
Comment (LucasYao93) [reply src/MySql/test/Az.MySql-TestResults.xml]: Done

File: src/MySql/custom/New-AzMySqlServer.ps1 (hunk 1 above)
Comment (LucasYao93) [reply src/MySql/custom/New-AzMySqlServer.ps1]: You're right,  The code is unnecessarily, The MinimalTlsVersion default value is TlsEnforcementDisabled. I have removed it.

Comment (msJinLei) [reply src/MySql/custom/New-AzMySqlServer.ps1]: > You're right, The code is unnecessarily, The MinimalTlsVersion default value is TlsEnforcementDisabled. I have removed it.

Please just remove something in "else {} ". Please check the customized change again

Comment (LucasYao93) [reply src/MySql/custom/New-AzMySqlServer.ps1]: I have only removed "else {} ". 
Thanks

**Review by msJinLei — APPROVED** [review]

**Pull request merged**